### PR TITLE
[16.0][MIG] account_bank_statement_chatter: Migration to 16.0

### PR DESCRIPTION
**Pull request by sergiobstoj**
Migration to 16.0 account_bank_statement_chatter module.

### CODE REVIEW

File: account_bank_statement_chatter/__manifest__.py [hunk 1]
```diff
@@ -3,13 +3,14 @@
...
 {
     "name": "Chatter on bank statements",
-    "version": "12.0.1.0.0",
+    "version": "16.0.1.0.0",
     "category": "Invoices & Payments",
     "license": "AGPL-3",
     "author": "Trey, " "Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/account-financial-tools",
     "depends": [
         "account",
```
Comment (aritzolea) [inline account_bank_statement_chatter/__manifest__.py]: La dependencia de `account` la puedes quitar, ya que `account_statement_base` ya depende de `account` también

**Review by aritzolea — APPROVED** [review]

**Review by MayteGLC — APPROVED** [review]
LGTM

**Comment by sergiobstoj** [comment]
@cubells I understand you were maintainer of this module on 12.0. Is this OK for you to merge?

**Comment by sergiobstoj** [comment]
@pedrobaeza is this Ok for merge?

**Comment by pedrobaeza** [comment]
Why not including this in `account_statement_base`?

**Comment by sergiobstoj** [comment]
@pedrobaeza Basically, in `account_statement_base`, model `account.bank.statement` is not inheriting from `mail.thread` and `mail.activity.mixin`.

I personally think is better to maintain the inheriting in this module.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Comment by pedrobaeza** [comment]
Please rebase and check pre-commit.

**Comment by sergiobstoj** [comment]
@pedrobaeza rebase and pre-commit applied

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**